muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/reducers/main-function/guider/index.ts

```diff
@@ -79,10 +79,70 @@ export interface GuiderNotificationStudentsDataType {
   assessmentrequest?: string;
 }
 
+/**
+ * ContactTypesArray for dropdowns etc.
+ */
+export const contactTypesArray = [
+  "OTHER",
+  "LETTER",
+  "EMAIL",
+  "PHONE",
+  "CHATLOG",
+  "SKYPE",
+  "FACE2FACE",
+  "ABSENCE",
+  "MUIKKU",
+] as const;
+
+/**
+ *  ContactTypes created from the ContactTypesArray
+ */
+export type ContactTypes = typeof contactTypesArray[number];
+
+/**
+ * IContactLogEvent
+ */
+export interface IContactLogEvent {
```

Comment:
I-prefixes in interface names are unnecessary. Interface names should be good enough to self explain what they are for without needing to indicate specifically that they are interfaces. Check all interfaces.